ArchiSteamFarm/Bot.cs

```diff
@@ -534,8 +534,17 @@ public sealed class Bot : IAsyncDisposable {
 					continue;
 				}
 
+				byte maxSetsAllowed = (byte) (Trading.MaxItemsPerTrade - result.Count);
```

Comment:
Since this method is a public API and plugins may use other value for maximum items per trade, can `Trading.MaxItemsPerTrade` be made an argument? (will require changing type from `byte` to `ushort`)

Reply:
Good idea